lib/Models/Catalog/CatalogReferences/MagdaReference.ts

```diff
@@ -863,8 +863,13 @@ const prepareDistributionFormat = createTransformer(
 
 function getAccessTypeFromMagdaRecord(magdaRecord: any): string {
   const record = toJS(magdaRecord);
-  const accessControl: any =
-    record && record.aspects && record.aspects["esri-access-control"];
-  const access = accessControl && accessControl.access;
-  return access;
+
+  // Magda V2 access control has higher priority.
+  if (record?.aspects?.["access-control"]) {
+    return record.aspects["access-control"].orgUnitId ? "non-public" : "public";
```

Comment:
should be `record.aspects["access-control"]?.orgUnitId`?

Reply:
Has tested `record.aspects["access-control"]` in the if statement.

Reply:
yes you are right 👍